### PR TITLE
Fixing CI failures

### PR DESCRIPTION
**Pull request by comius**
- Fixing CI instructions to include only complete targets
- Adding a dependency to rules_license
- Fixing gazelle reference
- Upgrading rules_go version

COAUTHOR=@kotlaja

### CODE REVIEW

File: .bazelci/postsubmit.yml [hunk 1]
```diff
@@ -1,22 +1,67 @@
...
     build_targets:
-    - "//..."
-    # test_targets:
-    # - "//..."
+    - "//src/..."
+    - "//test/..."
+    - "//android/..."
+    - "//rules/..."
+    - "-//src/java/com/example/sampleapp/..."
+    - "-//src/tools/ak/..."
```
Comment (comius) [inline .bazelci/postsubmit.yml:27]: Some subdirectories are missing on github.

File: .bazelci/postsubmit.yml [hunk 2]
```diff
@@ -1,22 +1,67 @@
...
+    bazel: last_green
     build_targets:
-    - "//..."
-    # test_targets:
-    # - "//..."
+    - "//src/..."
+    - "//test/..."
+    - "//android/..."
+    - "//rules/..."
+    - "-//src/java/com/example/sampleapp/..."
```
Comment (comius) [inline .bazelci/postsubmit.yml:26]: Has unexisting deps: "@androidsdk//com.android.support:appcompat-v7-25.0.0",
"@androidsdk//com.android.support:support-v4-25.2.0",

File: .bazelci/postsubmit.yml [hunk 3]
```diff
@@ -1,22 +1,67 @@
...
+    - "//test/..."
+    - "-//src/tools/..."
+    - "-//src/java/com/example/sampleapp/..."
   ubuntu1804:
+    bazel: last_green
     build_targets:
-    - "//..."
-    # test_targets:
-    # - "//..."
+    - "//src/..."
```
Comment (comius) [inline .bazelci/postsubmit.yml:22]: All the targets are included except //examples/..., which need to be build in a separate task, changing the directroy where their WORKSPACE file is.

The examples build works with Bazel@HEAD and using 2 extra flags:
`bazel build --experimental_google_legacy_api --experimental_enable_android_migration_apis //...`

**Comment by meteorcloudy** [comment]
I closed and reopened the PR to trigger the presubmit, you can check the result now

**Review by ted-xie — APPROVED** [review]
LGTM at a glance. Thanks! I'll try importing this internally and we can address any subsequent issues there.

File: .bazelci/postsubmit.yml (hunk 3 above)
Comment (ted-xie) [reply .bazelci/postsubmit.yml]: Yeah, I think we added these to the repo-level .bazelrc already.

**Pull request merged**